package.json

```diff
@@ -43,13 +43,15 @@
     "debug": "~2.1.1",
     "express": "~4.10.6",
     "glob": "^4.3.2",
-    "http-proxy": "^1.8.1",
     "jade": "~1.8.2",
     "js-yaml": "^3.2.5",
     "less-middleware": "1.0.x",
     "lodash": "^2.4.1",
     "morgan": "~1.5.1",
-    "serve-favicon": "~2.2.0"
+    "request": "^2.40.0",
+    "serve-favicon": "~2.2.0",
+    "semver": "^4.2.0",
```

Comment:
Doesn't look like the semver package is used here - holdover from your ongoing migrations work?